frontends/bmd/src/pages/print_page.test.tsx

```diff
@@ -1,20 +1,25 @@
 import { electionSample } from '@votingworks/fixtures';
 import { getBallotStyle, getContests, vote } from '@votingworks/types';
 import React from 'react';
+import { randomBase64 } from '@votingworks/utils';
 import { Route } from 'react-router-dom';
-import { mockOf } from '@votingworks/test-utils';
 import { render } from '../../test/test_utils';
-import { randomBase64 } from '../utils/random';
 import { PrintPage } from './print_page';
 import {
   electionSampleNoSealDefinition,
   electionSampleWithSealDefinition,
 } from '../data';
 
-// mock the random value so the snapshots match
-jest.mock('../utils/random');
-beforeEach(() => {
-  mockOf(randomBase64).mockReturnValue('CHhgYxfN5GeqnK8KaVOt1w');
+jest.mock('@votingworks/utils', (): { randomBase64: typeof randomBase64 } => {
+  const original = jest.requireActual<Record<string, unknown>>(
+    '@votingworks/utils'
+  );
+  // Mock random string generation so that snapshots match, while leaving the rest of the module
+  // intact
+  return {
+    ...original,
+    randomBase64: () => 'CHhgYxfN5GeqnK8KaVOt1w',
+  };
 });
```

Comment:
I tried a number of alternatives to the above to mock `randomBase64` while leaving the rest of `@votingworks/utils` intact. For example:

```ts
return {
  ...original,
  randomBase64: mockOf(randomBase64).mockReturnValue(
    'CHhgYxfN5GeqnK8KaVOt1w'
  ),
};
```

But this resulted in the following error:

```
The module factory of `jest.mock()` is not allowed to reference any out-of-scope variables.
Invalid variable access: randomBase64
Note: This is a precaution to guard against uninitialized mock variables. If it is ensured that the mock is required lazily, variable names prefixed with `mock` (case insensitive) are permitted.
```

I tried renaming variables as per the note in the error message. I also tried using `jest.doMock` instead of `jest.mock` (the latter hoists mock code above the relevant import while the former doesn't). But encountered various other issues.

I landed on what I did as the simplest working solution.